src/components/common/DrawerHeader.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { withStyles } from '@material-ui/core/styles';
+import Tooltip from '@material-ui/core/Tooltip';
+import IconButton from '@material-ui/core/IconButton';
+import { Online, Offline } from 'react-detect-offline';
+import Typography from '@material-ui/core/Typography/Typography';
+import CloudIcon from '@material-ui/icons/Cloud';
+import CloudOffIcon from '@material-ui/icons/CloudOff';
+import { withTranslation } from 'react-i18next';
+import ListItem from '@material-ui/core/ListItem';
+import ChevronLeftIcon from '@material-ui/icons/ChevronLeft';
+import ChevronRightIcon from '@material-ui/icons/ChevronRight';
+import ListItemIcon from '@material-ui/core/ListItemIcon';
+import ListItemSecondaryAction from '@material-ui/core/ListItemSecondaryAction';
+
+const styles = () => ({
+  drawerHeader: {
+    height: '55px',
```

Comment:
Is this the same in another places? Can we factor it out? Can this even change or would it break everything?